payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -13,6 +13,7 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.first
```

Comment:
What happens to `StripeGooglePayContract`?  It looks like this has some comparable functionality, but some things missing like settings the status bar color.

Reply:
I'll revisit `StripeGooglePayContract` after we ship. I think for now they can live side-by-side without impacting each other.